_drgn.pyi

```diff
@@ -202,13 +203,27 @@ class Program:
         :attr:`SymbolBinding.WEAK` symbol is found, it is returned. Otherwise,
         any matching symbol (e.g., :attr:`SymbolBinding.LOCAL`) is returned. If
```

Comment:
It occurs to me that these rules regarding symbol preferences seem to be only respected for name-based searches. Reading the comment of `dwfl_module_addrinfo` I don't see any documented preference in case multiple symbols match the address. Should I update that comment here? Or if you want to try to implement that preference scheme for address searches? (I could file an issue for that since it's out of scope here).

Reply:
It's undocumented, but `dwfl_module_addrinfo` does actually follow more or less the same rules; see the commit message for ff40f65f0db604254b47ae502cb8e3e2e2070444.